bin/node-template/README.md

```diff
@@ -1,93 +1,75 @@
 # Substrate Node Template
 
-[![Try on playground](https://img.shields.io/badge/Playground-Node_Template-brightgreen?logo=Parity%20Substrate)](https://docs.substrate.io/playground/) [![Matrix](https://img.shields.io/matrix/substrate-technical:matrix.org)](https://matrix.to/#/#substrate-technical:matrix.org)
+A fresh FRAME-based [Substrate](https://substrate.io/) node, ready for hacking :rocket:
+This is template is available from two locations:
 
-A fresh FRAME-based [Substrate](https://www.substrate.io/) node, ready for hacking :rocket:
+1. The maintained upstream version in the [Substrate repository](https://github.com/paritytech/substrate/tree/master/bin/node-template) that is updated as the rest of Substrate is developed.
```

Comment:
```suggestion
1. The upstream version in the [Substrate repository](https://github.com/paritytech/substrate/tree/master/bin/node-template) that is updated as the rest of Substrate is developed.
```

Reply:
I want to be explicit here, to make sure there is zero confusion that eh devhub template is not a place to make any PRs or issues. Just a handy template repo that they can use